### PR TITLE
dynamically add copyable step numbers

### PR DESCRIPTION
**Pull request by bakkot**
As an alternative to https://github.com/tc39/ecmarkup/pull/289, this PR tries to repair the regular copy-paste functionality. From discussion with the other ecma262 editors, we generally prefer this route: in particular the usual case is that you want to copy exactly one line, by triple-clicking it, and this serves that use-case better and without any new UI elements.

I'd [originally done this](https://github.com/tc39/ecmarkup/compare/e81364c..list-numbers) during the build rather than at runtime, but that turns out to make the rendered ecma262 spec 10% (600kb) larger. This seems better. It doesn't seem to noticeably affect time-to-interactive on the browsers I tested on.

### CODE REVIEW

File: js/listNumbers.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+var decimalBullet = Array.apply(null, Array(100)).map(function (a, i) {
```
Comment (ljharb) [inline js/listNumbers.js:1]: any reason not to use Array.from here rather than spreading a sparse array?

Comment (bakkot) [reply js/listNumbers.js:1]: This file is written in ES5. It's probably a good idea to switch over to more modern JS, but I don't want to that as part of this PR.

Comment (ljharb) [reply js/listNumbers.js:1]: This is a brand new file - do you mean the other files are written in ES5, and you wanted this one to match?

Comment (bakkot) [reply js/listNumbers.js:1]: Sorry, yes, that.

**Review by michaelficarra — APPROVED** [review]

**Pull request merged**